web/src/engine/keyboard/src/keyboards/kmxKeyboard.ts

```diff
@@ -0,0 +1,24 @@
+import { km_core_keyboard } from 'keyman/engine/core-processor';
+
+/**
+ * Acts as a wrapper class for KMX(+) Keyman keyboards
+ */
+export class KMXKeyboard {
+
+  constructor(id: string, keyboard: km_core_keyboard) {
+    this.id = id;
+    this.keyboard = keyboard;
+  }
+
+  id: string;
+  keyboard: km_core_keyboard;
```

Comment:
```suggestion
  constructor(public id: string, public keyboard: km_core_keyboard) {
  }
```

Also, consider making these private properties with read-only getters.

Reply:
done